sdks/python/apache_beam/runners/direct/executor.py

```diff
@@ -288,11 +295,16 @@ def call(self):
     scoped_metrics_container = ScopedMetricsContainer(metrics_container)
 
     for side_input in self._applied_ptransform.side_inputs:
+      # Find the projection of main's window onto the side input's window
```

Comment:
<!--new_thread; commit:0d82258b7674656d3a6f86748f47d23de8cf512f; resolved:0-->
Period at end of sentence.